subsystem/infra_env_test.go

```diff
@@ -163,6 +163,42 @@ var _ = Describe("Infra_Env", func() {
 		Expect(s.Size()).ShouldNot(Equal(0))
 	})
 
+	It("download infra-env files static network config file", func() {
+		By("Patching the infra env with a static network config")
+		netYaml := `interfaces:
+- ipv4:
+    address:
+    - ip: 192.0.2.1
+      prefix-length: 24
+    dhcp: false
+    enabled: true
+  name: eth0
+  state: up
+  type: ethernet`
+		staticNetworkConfig := models.HostStaticNetworkConfig{
+			NetworkYaml: netYaml,
+		}
+		staticNetworkConfigs := []*models.HostStaticNetworkConfig{&staticNetworkConfig}
+		updateParams := &installer.UpdateInfraEnvParams{
+			InfraEnvID: infraEnvID,
+			InfraEnvUpdateParams: &models.InfraEnvUpdateParams{
+				StaticNetworkConfig: staticNetworkConfigs,
+			},
+		}
+		_, err := userBMClient.Installer.UpdateInfraEnv(ctx, updateParams)
+		Expect(err).NotTo(HaveOccurred())
+		By("Downloading the static network config archive")
+		buf := &bytes.Buffer{}
+		_, err = userBMClient.Installer.V2DownloadInfraEnvFiles(ctx, &installer.V2DownloadInfraEnvFilesParams{InfraEnvID: infraEnvID, FileName: "static-network-config"}, buf)
+		Expect(err).NotTo(HaveOccurred())
+		By("Verifying the contents of the archive")
+		contents := buf.String()
+		Expect(len(contents)).ShouldNot(Equal(0))
+		Expect(contents).To(ContainSubstring("/etc/assisted/network/host0"))
```

Comment:
Aren't these contents an archive?
Do we need to unpack this before doing these kinds of comparisons or will these strings always exist even in the archive directly?

Reply:
I suppose this also applies to the unit test

Reply:
It is an archive. 
When I print out the archive string (from the unit test) it looks like this and contains these strings:
```
/etc/assisted/network/host10000600000000000000000000000005000000000000014313 0ustar0000000000000000static network config data of first host
```
Is it better to always unpack it? 

I thought it only needed to be unpacked if it was compressed since the bytes need to be transformed?

Reply:
:man_shrugging: I suppose it works and the tar format isn't likely to change so I'm fine with it.

Reply:
Thanks for bringing it up! I'll keep an eye out for unit test failures regarding it if it does change in the future